themes/modern-t/modern-t.theme.bash

```diff
@@ -1,3 +1,10 @@
+# The "modern-t" theme is a "modern" theme variant with support
+# for "t", the minimalist python todo list utility by Steve Losh.
+# Get and install "t" at https://github.com/sjl/t#installing-t
+#
+# Warning: The Bash-it plugin "todo.plugin" breaks the "t"
```

Comment:
I think I prefer to fix the plugin and not add a warning here. I will try to fix it up in the upcoming days

Reply:
Sure. I think "t" is pretty well _unknown_, and unless I missed it, I see no readme for the "modern" themes.  That would be a better place for this info to be.  Would a readme doc for the modern themes be appreciated? I have come this far, I'd like to try to get a PR in if I can.  I would think it should be here "bash-it/docs/themes-list/".  I also noticed the screenshots for "modern-t" also show that "t" is not installed, as there is an error. This is kind of what I mean, most that might try modern-t have no idea that there is an actual dependence called "t", or even if they figure it out, it's not easy to find, sadly as it is a very good little utility. 

Also, It would be easy to create a "todo.sh" variant, "modern-todo" with the same CLI integration functionality as "modern-t". I'd be happy to submit that one, but I am unsure how people feel about having multiple varients of the same theme.

Reply:
I think adding a readme doc will be a great idea of you can do it- the location you assumed is the correct one as well!
I do not think we need another theme for that, but we might need to improve (and add) docs about "modern-t" and also maybe add a check that it indeed uses the "t" utility